XamlControlsGallery/ControlPages/NavigationViewPage.xaml.cs

```diff
@@ -171,7 +172,24 @@ private void NavigationView_SelectionChanged7(Microsoft.UI.Xaml.Controls.Navigat
                 contentFrame7.Navigate(pageType, null, args.RecommendedNavigationTransitionInfo);
             }
         }
- 
+
+        private void NavigationView_SelectionChanged8(Microsoft.UI.Xaml.Controls.NavigationView sender, Microsoft.UI.Xaml.Controls.NavigationViewSelectionChangedEventArgs args)
+        {
+            if (args.IsSettingsSelected)
```

Comment:
This code looks copy+pasted from previous SelectionChanged handlers. Do we need so many - or is it possible to refactor this into a more straightforward chunk of shared code?

Reply:
Because the content frames aren't a property of the NavigationView, I don't see a way to combine these into one SelectionChanged handler when the sender can't give us info on which frame to update. @ojhad let me know if you have an easier solution to this.

Reply:
No worries if this can't be easily refactored.